src/panoptes/pocs/camera/camera.py

```diff
@@ -131,6 +131,15 @@ def __init__(self,
                 # Keep a list of active subcomponents
                 self.subcomponents[attr_name] = subcomponent
 
+        # Apply the initial focus offset
+        # This is required so that the focuser initial position corresponds to focus_offset=0
+        if self.has_focuser and self.has_filterwheel:
+            current_filter = self.filterwheel.current_filter
+            focus_offset = self.filterwheel.focus_offsets.get(current_filter, 0)
+            self.logger.debug(f"Initial focus offset for {current_filter} filter: {focus_offset}")
+            if focus_offset:
```

Comment:
Won't this always have a value now and so the if check isn't necessary?

Reply:
Or do you want to skip the call if the value is zero?

Reply:
@wtgee Yeah, that was just to skip the call if the value was 0, to maximise efficiency. Can change it if you want?

Reply:
Ok with me. I'll merge.